components/core/fileminio.py

```diff
@@ -0,0 +1,40 @@
+from flask import Response, request
+from DownloadDaemon import *
+from minio import Minio
+from minio.error import ResponseError
+from DownloadManager import *
+from DBCon import *
+import sys
+import sqlalchemy.pool as pool
+
+client = Minio('localhost:9000', secure=False,
+               access_key='bassa',
```

Comment:
already working on this credentials thing

> @vivonk @agentmilindu if we are integrating minio with @VibhorCodecianGupta file management work. We should make better use of minio client SDK. It's better to fetch files from minio server rather using a separate client (minio browser) for showing up the files as it seems to be a bad practice. What are your thoughts?
> @demon-36

@kmehant  Minio is going to work both ways - it will be accessible to admin to visit Minio and view downloaded files and when user will request to download files from completed section then it will get file from Minio server and serve to user.
these changes are going to be added in next commits.

Reply:
already working on this thing :)

Reply:
One problem to resolve afterwards : 
@kmehant @demon-36  Minio generate access_key and secret_key randomly on each start which we have to add to client config manually or another way is keep a static pair of secret_key and access_key and add it to Minio and client for complete automated process. For first release, we have to make the things static, after full feature release we will move to automate the setting up keys.
When @demon-36 @kmehant you guys move to K8S, keep this point to discussion.